src/main/webapp/pages/projects/project_samples.html

```diff
@@ -4,47 +4,53 @@
   xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
```

Comment:
Why so many formatting changes?  You are going to create a huge headache for me when I merge this with my share samples branch.

Reply:
Sorry, editor troubles... please see e50b2deee76325179cb51cc1d63b5119a5b12610.